app/src/main/AndroidManifest.xml

```diff
@@ -133,7 +133,8 @@
 
         <activity
             android:name=".review.ReviewActivity"
-            android:label="@string/title_activity_review" />
+            android:label="@string/title_activity_review"
+            android:configChanges="orientation|screenSize|keyboard"/>
 
```

Comment:
This technique is not suggested (see: https://developer.android.com/guide/topics/resources/runtime-changes ), and I don't think the reason of this bug requires this changes. Can you try to solve this by using savedinstancestate?